sourced/ml/__main__.py

```diff
@@ -68,6 +107,27 @@ def get_parser() -> argparse.ArgumentParser:
     # Create and construct subparsers
 
     subparsers = parser.add_subparsers(help="Commands", dest="command")
+
+    repo2coocc_parser = subparsers.add_parser(
+        "repos2coocc", help="Produce the co-occurrence matrix from a Git repository.",
+        formatter_class=ArgumentDefaultsHelpFormatterNoNone)
+    add_engine_args(repo2coocc_parser)
+
+    repo2coocc_parser.add_argument(
```

Comment:
I bet we should move `-r`, `--min-docfreq` and `-l` to a separate group of arguments to reuse in other commands.

Reply:
I missed that, but I think it is better to use `"-i", "--input"` instead of `"-r", "--repositories"`. What do you think, @vmarkovtsev?